### PR TITLE
[Fix][Transforms-V2] Fix LLM transform can not parse boolean value type

### PR DESCRIPTION
**Pull request by hawk9821**
### Purpose of this pull request

issues:  #7617 

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

TestLLMBooleanIT#testLLMWithOpenAI

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by Hisoka-X — COMMENTED** [review]
Why not change it in https://github.com/apache/seatunnel/blob/b35bd94bfb974f56e7198d3d054cbeddf0762c64/seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/llm/LLMTransform.java#L126 ?

**Comment by hawk9821** [comment]
> Why not change it in
> 
> https://github.com/apache/seatunnel/blob/b35bd94bfb974f56e7198d3d054cbeddf0762c64/seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/llm/LLMTransform.java#L126
> 
> ?

LLM return [True],  The JSON standard requires Boolean values to be lowercase true or false, not uppercase True or False.
`OBJECT_MAPPER.readValue("[True]", new TypeReference<List<String>>() {});`
resulting in Json serialization errors.

**Comment by Hisoka-X** [comment]
> > Why not change it in
> > https://github.com/apache/seatunnel/blob/b35bd94bfb974f56e7198d3d054cbeddf0762c64/seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/nlpmodel/llm/LLMTransform.java#L126
> > 
> > ?
> 
> LLM return [True], The JSON standard requires Boolean values to be lowercase true or false, not uppercase True or False. `OBJECT_MAPPER.readValue("[True]", new TypeReference<List<String>>() {});` resulting in Json serialization errors.

Got it

File: seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-1/src/test/java/org/apache/seatunnel/e2e/transform/TestLLMBooleanIT.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.testcontainers.utility.DockerLoggerFactory;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class TestLLMBooleanIT extends TestSuiteBase implements TestResource {
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-1/src/test/java/org/apache/seatunnel/e2e/transform/TestLLMBooleanIT.java]: Please reuse `TestLLMIT`.

Comment (hawk9821) [reply seatunnel-e2e/seatunnel-transforms-v2-e2e/seatunnel-transforms-v2-e2e-part-1/src/test/java/org/apache/seatunnel/e2e/transform/TestLLMBooleanIT.java]: get

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @hawk9821

**Review by hailin0 — APPROVED** [review]

**Pull request merged**